[2023-06-15T02:35:26.702+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_ingestion_gcs_dag.convert_to_parquet_task manual__2023-06-15T02:23:53.082621+00:00 [queued]>
[2023-06-15T02:35:26.720+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_ingestion_gcs_dag.convert_to_parquet_task manual__2023-06-15T02:23:53.082621+00:00 [queued]>
[2023-06-15T02:35:26.721+0000] {taskinstance.py:1282} INFO - 
--------------------------------------------------------------------------------
[2023-06-15T02:35:26.722+0000] {taskinstance.py:1283} INFO - Starting attempt 1 of 2
[2023-06-15T02:35:26.723+0000] {taskinstance.py:1284} INFO - 
--------------------------------------------------------------------------------
[2023-06-15T02:35:26.744+0000] {taskinstance.py:1303} INFO - Executing <Task(PythonOperator): convert_to_parquet_task> on 2023-06-15 02:23:53.082621+00:00
[2023-06-15T02:35:26.765+0000] {standard_task_runner.py:55} INFO - Started process 12832 to run task
[2023-06-15T02:35:26.772+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag', 'convert_to_parquet_task', 'manual__2023-06-15T02:23:53.082621+00:00', '--job-id', '261', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmp7pupec_p']
[2023-06-15T02:35:26.777+0000] {standard_task_runner.py:83} INFO - Job 261: Subtask convert_to_parquet_task
[2023-06-15T02:35:26.915+0000] {task_command.py:388} INFO - Running <TaskInstance: data_ingestion_gcs_dag.convert_to_parquet_task manual__2023-06-15T02:23:53.082621+00:00 [running]> on host 570ce802d94a
[2023-06-15T02:35:27.038+0000] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag
AIRFLOW_CTX_TASK_ID=convert_to_parquet_task
AIRFLOW_CTX_EXECUTION_DATE=2023-06-15T02:23:53.082621+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-06-15T02:23:53.082621+00:00
[2023-06-15T02:35:27.041+0000] {taskinstance.py:1775} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/data_ingestion_gcs_dag.py", line 25, in convert_to_parquet
    csv_files = glob.glob(os.path.join(dataset_path, "*.csv"))
NameError: name 'glob' is not defined
[2023-06-15T02:35:27.059+0000] {taskinstance.py:1326} INFO - Marking task as UP_FOR_RETRY. dag_id=data_ingestion_gcs_dag, task_id=convert_to_parquet_task, execution_date=20230615T022353, start_date=20230615T023526, end_date=20230615T023527
[2023-06-15T02:35:27.081+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 261 for task convert_to_parquet_task (name 'glob' is not defined; 12832)
[2023-06-15T02:35:27.107+0000] {local_task_job.py:212} INFO - Task exited with return code 1
[2023-06-15T02:35:27.143+0000] {taskinstance.py:2585} INFO - 0 downstream tasks scheduled from follow-on schedule check
